purchases/src/main/kotlin/com/revenuecat/purchases/models/StoreProduct.kt

```diff
@@ -91,6 +93,18 @@ interface StoreProduct {
      */
     fun copyWithOfferingId(offeringId: String): StoreProduct
 
+    /**
+     * Null for INAPP products. The price of the [StoreProduct] in the given locale in a weekly recurrence.
+     * This means that, for example, if the period is monthly, the price will be divided by 4.
+     * It uses a currency formatter to format the price in the given locale.
+     * Note that this value may be an approximation.
+     * For Google subscriptions, this value will use the basePlan to calculate the value.
+     * @param locale Locale to use for formatting the price. Default is the system default locale.
+     */
+    fun pricePerWeek(locale: Locale = Locale.getDefault()): Price? {
```

Comment:
We should add a `@JvmOverloads` here I think, so thee default value works in Java. Same in `pricePerYear`

Reply:
Oh it's because `@JvmOverloads` isn't allowed on `interface` methods.

Reply:
We can add those explicitly overloaded in a separate PR.